contracts/exchange-forwarder/package.json

```diff
@@ -14,12 +14,11 @@
         "build:ts": "tsc -b",
         "build:ci": "yarn build",
         "pre_build": "run-s compile contracts:gen generate_contract_wrappers",
-        "test": "yarn run_mocha",
+        "test": "mocha --require source-map-support/register --require make-promises-safe '../integrations/lib/test/forwarder/**/*.js' --timeout 100000 --bail --exit",
```

Comment:
won't this mean the forwarder tests will run twice on CI?

Reply:
ah yeah, forgot about that